### PR TITLE
Replace playbooks/upgrades/upgrade.yml

### PR DESCRIPTION
**Pull request by mikenairn**
## Additional Information
Replace playbooks/upgrades/upgrade.yml with symlink to playbooks/upgrade.yml

Related:

https://github.com/integr8ly/installation/pull/1246

### CODE REVIEW

File: playbooks/upgrades/upgrade.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+../upgrade.yml
```
Comment (pmccarthy) [inline playbooks/upgrades/upgrade.yml]: If this is just a symlink, why are we keeping this file? Tower does reference it but we can update the playbook path here: https://github.com/integr8ly/ansible-tower-configuration/blob/master/playbooks/roles/integreatly/tasks/bootstrap_osd_update.yml#L84

Also didn't realise that playbook path is hardcoded in the tower-config so will need to make this a variable, not relevant here though.

**Comment by mikenairn** [comment]
/hold cancel

@pb82 @pmccarthy Can we get an lgtm on this please.

**Pull request merged**